src/ZiraLink.Api.Application/Tools/Tools.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.NetworkInformation;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace ZiraLink.Api.Application.Tools
+{
+    public class Tools : ITools
```

Comment:
Tools is the namespace, another name like HttpTools would be a suitable name

Reply:
Done